src/protocol/deserialize_auto.cpp

```diff
@@ -2576,14 +2576,19 @@ namespace OpcUa
 
 
     template<>
+    void DataDeserializer::Deserialize<CallRequestParameters>(CallRequestParameters& data)
+    {
+        DeserializeContainer(*this, data.MethodsToCall);
+    }
+
+    template<>
     void DataDeserializer::Deserialize<CallRequest>(CallRequest& data)
```

Comment:
Directly modified it. :-( Sorry! Ok, I will change it. Btw great software design/structure! I like it!